docs/operations/vercel-environment-setup.md

```
@@ -0,0 +1,161 @@
+
+### Step 2: Add Preview Deploy Key
+
+**⚠️ IMPORTANT**: This must be set for **Preview environment only**
+
+```
+Key: CONVEX_DEPLOY_KEY
+Value: preview:phaedrus:scry|eyJ2MiI6IjhmZmZjNjhkYmIwMjRmODY4NTI1ZWRkY2VlZTg5OWY2In0=
+Environment: ✓ Preview (ONLY - uncheck Production and Development)
+```
+
+**Click "Save"**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)